operations/system_admin_toolkit/usage/SAT_and_IUF.md

```diff
@@ -168,57 +141,46 @@ example showing how to find the IUF activity.
    cp -pv "${ACTIVITY_DIR}/state/session_vars.yaml" "${BOOTPREP_DIR}"
    ```
 
-1. (`ncn-m001#`) Modify the CSM version in the copied `session_vars.yaml`:
+1. (`ncn-m001#`) Before using this file, it is recommended to modify the
+   `default.suffix` value in the copied `session_vars.yaml`.
 
-   ```bash
-   yq w -i "${BOOTPREP_DIR}/session_vars.yaml" 'csm.version' "${CSM_RELEASE}"
-   ```
+   As long as the `sat bootprep` input file uses `{{default.suffix}}` in the
+   names of the CFS configurations and IMS images, this will ensure new CFS
+   configurations and IMS images are created with different names from the ones
+   created in the IUF activity.
 
-1. (`ncn-m001#`) Update the `working_branch` if one is used for the CSM product.
-
-   By default, a `working_branch` is not used for the CSM product. Check if there is a
-   `working_branch` specified for CSM:
+   The example below sets the suffix to "-debug":
 
    ```bash
-   yq r "${BOOTPREP_DIR}/session_vars.yaml" 'csm.working_branch'
-   ```
-
-   If this produces no output, a `working_branch` is not in use for the CSM product, and this step
-   can be skipped. Otherwise, it shows the name of the working branch. For example:
-
-   ```text
-   integration-1.4.0
+   yq w -i -- "${BOOTPREP_DIR}/session_vars.yaml" 'default.suffix' "-debug"
    ```
 
-   In this case, be sure to manually update the version string in the working branch to match the
-   new working branch. Then check it again. For example:
+1. (`ncn-m001#`) Change directory to the `BOOTPREP_DIR` and run `sat bootprep`,
+   passing the `session_vars.yaml` file as the value of the `--vars-file`
+   option.
 
+   This example uses the `management-bootprep.yaml` file:
+   
    ```bash
-   yq w -i "${BOOTPREP_DIR}/session_vars.yaml" 'csm.working_branch' "integration-${CSM_RELEASE}"
-   yq r "${BOOTPREP_DIR}/session_vars.yaml" 'csm.working_branch'
+   cd "${BOOTPREP_DIR}"
+   sat bootprep run --vars-file session_vars.yaml management-bootprep.yaml
    ```
```

Comment:
Should this piece be put in the next section? It seem more applicable to building images and cfs configs than it does to "Obtaining IUF session variables and bootprep files".

Reply:
Yeah, I think that's fair. I just wanted to make sure it was clear what you would do with these files in this section to actually use them in bootprep.

I will move it perhaps with a comment about generally how these files would be used.